language-extensions/dotnet-core-CSharp/build/windows/create-dotnet-core-CSharp-extension-zip.cmd

```diff
@@ -24,18 +24,18 @@ REM Delete the ref folder so that the zip can be loaded by the SPEES
 RD /S /Q %BUILD_OUTPUT%\ref
 POPD
 
+REM Define files to compress, conditionally including .pdb files if BUILD_CONFIGURATION is "debug"
+SET FILES_TO_COMPRESS=%BUILD_OUTPUT%\*.dll, %BUILD_OUTPUT%\Microsoft.SqlServer.CSharpExtension.runtimeconfig.json, %BUILD_OUTPUT%\Microsoft.SqlServer.CSharpExtension.deps.json
+IF /I "%BUILD_CONFIGURATION%"=="debug" (SET FILES_TO_COMPRESS=%FILES_TO_COMPRESS%, %BUILD_OUTPUT%\*.pdb)
+
 REM Package the signed binaries.
 REM
-powershell -NoProfile -ExecutionPolicy Unrestricted ^
- -Command "Compress-Archive -Force -Path %BUILD_OUTPUT%\*.dll, %BUILD_OUTPUT%\*.pdb, `" ^
- "%BUILD_OUTPUT%\Microsoft.SqlServer.CSharpExtension.runtimeconfig.json, `" ^
- "%BUILD_OUTPUT%\Microsoft.SqlServer.CSharpExtension.deps.json `" ^
- "-DestinationPath %BUILD_OUTPUT%\packages\dotnet-core-CSharp-lang-extension.zip"
+powershell -NoProfile -ExecutionPolicy Unrestricted -Command "Compress-Archive -Force -Path %FILES_TO_COMPRESS% -DestinationPath %BUILD_OUTPUT%\packages\dotnet-core-CSharp-lang-extension.zip"
```

Comment:
can we split the command into multiple lines for better readability using `^`  - line limit should be 100 chars